creator-node/src/services/prometheusMonitoring/prometheus.constants.js

```diff
@@ -45,10 +96,26 @@ const Metrics = Object.freeze({
       labelNames: ['code'],
       buckets: [0.1, 0.3, 0.5, 1, 3, 5, 10] // 0.1 to 10 seconds
     }
+  },
+  [MetricNames.ISSUE_SYNC_SECONDS_WAITING_FOR_COMPLETION_HISTOGRAM]: {
+    metricType: MetricTypes.HISTOGRAM,
+    metricConfig: {
+      name: MetricNames.ISSUE_SYNC_SECONDS_WAITING_FOR_COMPLETION_HISTOGRAM,
+      help: 'Seconds spent monitoring an outgoing sync request issued by this node to be completed (successfully or not)',
+      labelNames:
+        MetricLabelNames.ISSUE_SYNC_SECONDS_WAITING_FOR_COMPLETION_HISTOGRAM,
+      // 5 buckets in the range of 1 second to max seconds before timing out a sync request
+      buckets: exponentialBucketsRange(
+        1,
+        config.get('maxSyncMonitoringDurationInMs') / 1000,
```

Comment:
also can we truncate the bucket values? either to int or some reasonable # of decimals

Reply:
it's already going to be an int because it's just an int amount of millis divided by 1000 to make it into seconds (unless millis is <1000, which would be an unreasonable env var value in this case). I'll definitely truncate -- the number of decimals is kind of ridiculous now that I look at the results